json-rpc/src/runtime.rs

```diff
@@ -437,7 +437,8 @@ async fn rpc_request_handler(
                                 request_type_label,
                                 name,
                                 LABEL_SUCCESS,
-                                sdk_lang.as_str(),
+                                &sdk_info.language.as_str(),
```

Comment:
You should remove the leading `&` on this line (the one below is needed though). Same goes for all these similar lines.